src/app/core/components/directives/details-height/details-height.directive.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  Directive, ElementRef, Inject, Input, OnDestroy, OnInit,
+} from '@angular/core';
+import { WINDOW } from 'app/helpers/window.helper';
+
+const headingHeightInitialDefault = 213;
+const headingHeightDefault = 137;
+
+///
+/// This directive is used to dynamically adjust height of the "details" block in a "master-details" layout
+/// to fill the bottom space, which becomes available when user scrolls the page down,
+/// so the page's heading is shifted off the screen
+///
```

Comment:
Prefer jsdoc style
```
/**
 *
 */
```

Reply:
This is addressed in one of the latest commits